DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
We should probably target `netstandard2.0` for this project like we do for the Durable extension. That will make sure there isn't any weird behavior when customers are using packages that target two different netstandard frameworks.